H_CZ_2_2_complete_ECC_set_modified.json

```diff
@@ -1,11 +1,171 @@
-[[],
-{
-"0_5": [
-[[2,0,0,0,["da715f84238f"],[4.31710838500297678e-01,2.10647530283869677e-01]],[]]
-,[[2,0,0,2,["da715f84238f"],[4.31710838500297567e-01,2.10647530283869650e-01]],[["h", ["Q1"],["Q1"]],["h", ["Q1"],["Q1"]]]]
-,[[2,0,0,2,["da715f84238f"],[4.31710838500297622e-01,2.10647530283869677e-01]],[["h", ["Q0"],["Q0"]],["h", ["Q0"],["Q0"]]]]
-,[[2,0,0,2,["da715f84238f"],[4.31710838500297678e-01,2.10647530283869677e-01]],[["cz", ["Q0", "Q1"],["Q0", "Q1"]],["cz", ["Q0", "Q1"],["Q0", "Q1"]]]]
-,[[2,0,0,2,["da715f84238f"],[4.31710838500297678e-01,2.10647530283869677e-01]],[["cz", ["Q0", "Q1"],["Q0", "Q1"]],["cz", ["Q1", "Q0"],["Q1", "Q0"]]]]
-]
-}
+[
```

Comment:
Oh, I should not format this file, it is a mistake, and I will recover it. Actually I skipped the JSON file in the `pre-commit` configuration [here](https://github.com/quantum-compiler/quartz/blob/e3f6226be65efa5f07df4029bc99f8509c8b39db/.pre-commit-config.yaml), but this file is changed before I add this config.

Reply:
I added a `.clang_format` file in the root path of our repo by 05f7f62 , in which I directly use Google style.

Reply:
I realized the issue of different `clang-format` version you've mentioned, so I update the `.clang_format` of Google style with concrete content generated by `clang-format version 14.0.6` on my machine.

Reply:
BTW, in the pre-commit [configuration](https://github.com/quantum-compiler/quartz/blob/e3f6226be65efa5f07df4029bc99f8509c8b39db/.pre-commit-config.yaml), the python files will be formatted by https://github.com/psf/black instead of `clang-format`.